acceptance/shared/.kitchen.digitalocean.yml

```diff
@@ -9,6 +9,12 @@ driver:
   transport:
     ssh_key: <%= ENV['DIGITALOCEAN_SSH_KEY_PATH'] %>
 
+provisioner:
+  name: chef_zero
+  product_name: chef
+  product_version: latest
+  channel: current
```

Comment:
We're not currently supporting digital ocean so we haven't changed this file - it's still just a placeholder.